gke/provision-gke.sh

```diff
@@ -36,11 +36,11 @@ else
         --num-nodes "4" \
         --enable-autoscaling --min-nodes 4 --max-nodes 8 \
         --network "default" \
-        --enable-cloud-logging \
-        --enable-cloud-monitoring \
         --enable-ip-alias \
         --cluster-version=${CLUSTER_VERSION} \
-        --enable-stackdriver-kubernetes
+        --enable-stackdriver-kubernetes \
+        --identity-namespace=${PROJECT}.svc.id.goog \
```

Comment:
I personally think these 2 should be in the CSM script so we separate this from general cluster creation. What do you think?

Reply:
The label could be part of the CSM script, but since we didn't updated the CSM (or ASM now) part yet and this is required, it's better to leave the label here . Regarding workload identity, it takes around 20 min per node to enable it later, so it's better to create with it from the start.